src/components/InfoCardSection/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { InfoCard, TeamItem } from '../../components';
+import { missionContent, dataSourceContent } from '../../constants';
+
+const InfoCardSection = () => (
+  <div className="info-cards">
+    <InfoCard title="Mission">
+      <div
+        dangerouslySetInnerHTML={{ __html: missionContent.text }}
```

Comment:
I dig the constants file were using and its ability for us to easily update the site. However, when I first started looking through the code base I found it kinda hard to find where certain things were being rendered because of the level of indirection the constants file creates and the number of components we have herein. This is likely just a side effect of me not knowing the code base 😛.

However, from what I can tell the new redesign is mostly just a static page with a single graph. Do you think we it would make it easier to understand if we just inlined the static content here in? That way we wouldn't have to do any sort of dangerouslySetInnerHTML here either.

Reply:
in regards to the constants file, on Gitter @stephanieinez recently mentioned using Contentful as a CMS to handle content. Thoughts on this @pwhisenhunt?

Reply:
Oh, that's an interesting idea, I must have overlooked that on Gitter.

I should preface that I haven't thoroughly looked through the full mockups of the redesign so there may be things that I'm missing. Yeah, we could definitely go down the route of using a CMS. However, I'm not sure if the current redesign merits a full CMS as it looks like it's going to be a mostly static page, minus the graph. If we have the goal of continually adding content to the website I think a CMS may be merited although the constants file is pretty easy to update.

Lastly, I think it really just depends on how much we plan on updating and modifying the website and what problems the CMS aims to will solve; e.g. maybe a CMS would make updates more accessible to folks not familiar with github but maybe we could outline how to update the site in the readme instead?

Reply:
I think we need to identify:
a) how often this content is going to be updated
b) who is going to be doing the updating 

I've been thinking that you're right, maybe using a CMS might be a bit OTT for this project. I've inlined the HTML in this file as you suggested, but I also think this looks a but messy - what do you think?

Reply:
Yeah, great questions @stephanieinez. Yeah, I'm not sure of the answers to those questions. I vote that we punt on a CMS until updates because strenuous.

Yeah, you're definitely right that it doesn't quiet look as elegant but I kinda think it's a bit easier to read and understand since you can see directly in the markup where the copy is and what not.